src/ONE.Models.CSharp/Enterprise/Twin/InstrumentCategoryConstants.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+
+namespace ONE.Models.CSharp.Enterprise.Twin
+{
+    public static class InstrumentCategoryConstants
+    {
+        public const int Id = 3;
+        public static class ClientIngestType
+        {
+            public const string RefId = "9c77b2dc-1c6b-4f02-88e1-eac43692d1c0";
+            public static readonly Guid Guid = new Guid(RefId);
+            public static class ClientIngestWindowsServiceSubType
+            {
+                public const string RefId = "02d6fd74-f610-428f-8dd6-2789960e7fd7";
+                public static readonly Guid Guid = new Guid(RefId);
+            }
+        }
+        public static class ClientUtilityType
+        {
+            public const string RefId = "21b4d9dc-48e0-4717-a487-6e92e0d8fcdb";
+            public static readonly Guid Guid = new Guid(RefId);
+            public static class WimsDataConversionServiceSubType
+            {
+                public const string RefId = "51cf4cd3-962a-48fb-a092-ae9fa003ca59";
+                public static readonly Guid Guid = new Guid(RefId);
+            }
+        }
+        public static class DataSourceDriverType
+        {
+            public const string RefId = "3209ac8f-640b-4f68-a74b-08b670187182";
+            public static readonly Guid Guid = new Guid(RefId);
+            public static class DataSourceDriverTest
+            {
+                public const string RefId = "cafd1356-7e06-44d9-9f28-3a897fbc4cf8";
+                public static readonly Guid Guid = new Guid(RefId);
+            }
+            public static class DataSourceDriverOpenWeatherMap
```

Comment:
we deleted this driver in https://github.com/AquaticInformatics/aquarius-one-ingest/pull/37  not sure if it means this class can/should be removed

Reply:
I would be reticent to revoke the GUID assigned.  This could be a little working ahead.

Reply:
Noted